.github/workflows/ci.yml

```diff
@@ -601,9 +590,6 @@ jobs:
         env:
           REDWOOD_TEST_PROJECT_PATH: '${{ steps.set-up-test-project.outputs.test-project-path }}'
           REDWOOD_DISABLE_TELEMETRY: 1
-          # Temp workaround for Replay specific failing tests
-          DISABLE_REPLAY_UPLOAD: 1
```

Comment:
Does the fallback API key fix these tests?!

Reply:
Ah no, this is just cleanup (unrelated), since we don't use replay-browser at all for these tests, there's no point with `DISABLE_REPLAY_UPLOAD`, since it's not recorded.

Reply:
Can you please create a separate PR for that?

Reply:
Sure yeah, one sec.

Reply:
Done.

Reply:
Thank you